configs/brvs/application.properties

```diff
@@ -0,0 +1,14 @@
+credential.accountId=brvs@brvs
+credential.pubkey=b9679bbf526a1c936cd1144b56a370d376fa8246b248cd72f952b45a2f20bdad
+credential.privkey=56a3c52cd039d9b73a1720052600a20962350b1ea169b4783cefbf87ed99406a
```

Comment:
It is not secure to store private key in configs.

Reply:
what flexible mechanism of brvs key pair provision would you suggest?

Reply:
It's ok for test keys. But for production configs should be added as environment variables on startup

Reply:
You can put your keys in `deploy/iroha/keys` and provide a path to these keys instead. This is how we do it.